<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAbout</class>
 <widget class="QDialog" name="DialogAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>543</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ToolSimplifyNewick.qrc">
    <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_simplify_newick_version">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SimplifyNewick version: ?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_copyright">
     <property name="text">
      <string>Copyright (C) 2010 Richel Bilderbeek</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_author">
     <property name="text">
      <string>Programmed by Richel Bilderbeek </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_date">
     <property name="text">
      <string>on the 16th September 2010</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_download">
     <property name="text">
      <string>SimplifyNewick can be downloaded from</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_url">
     <property name="text">
      <string>http://www.richelbilderbeek.nl/ToolSimplifyNewick.htm</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_licence_2">
     <property name="text">
      <string>Licenced under GPL 3.0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_other_libraries">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Other libraries used</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_boost_version">
     <property name="text">
      <string>Boost version: ?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_cln_version">
     <property name="text">
      <string>CLN version: ?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_newick_vector_version">
     <property name="text">
      <string>NewickVector version: ?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_licence">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Licence</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_whats_new">
        <property name="text">
         <string>&amp;What's new?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_about_qt">
        <property name="text">
         <string>About &amp;Qt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="text_licence">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>SimplifyNewick, tool to randomly simplify Newick phylogenies
Copyright (C) 2010 Richel Bilderbeek

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
GNU General Public License for more details.
You should have received a copy of the GNU General Public License
along with this program.If not, see &lt;http://www.gnu.org/licenses/&gt;.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ToolSimplifyNewick.qrc"/>
 </resources>
 <connections/>
</ui>
